globus_action_provider_tools/data_types.py

```diff
@@ -105,18 +104,17 @@ class ActionRequest(BaseModel):
         None,
         description="A timestamp indicating by which time the action must complete. The request may be rejected if the Action Provider does not expect to be able to complete the action before the deadline or if it represents a time greater than the maximum_deadline specified in the Provider Description.",
     )
-    release_after: Optional[str] = Field(
+    release_after: Optional[datetime.timedelta] = Field(
```

Comment:
This is excellent. We have not been consistent on this, and pydantic seems to unify this whole thing. Most happy about this!!!